assets/sass/_split-section.scss

```diff
@@ -0,0 +1,41 @@
+.split-section {
```

Comment:
I suspect we can remove this and just stick with the 50/50 split we have now. I'm all for exploring alternate layouts, but I don't think this is something we _need_ to do in this PR.